install-dependencies.sh

```diff
@@ -52,6 +52,9 @@ debian_base_packages=(
 
 fedora_packages=(
     clang
+    compiler-rt
```

Comment:
`compiler-rt` is a indirect weak (Recommended by clang-libs) of clang. as per https://src.fedoraproject.org/rpms/clang/blob/rawhide/f/clang.spec#_167 .

> Install compiler-rt if you want the Blocks C language extension or to
> enable sanitization and profiling options when building

we should not rely on user or packager management tool's default settings to install the "Recommended" packages. so probably we need to have this dependency in seastar's `install-dependencies.sh` ? i can create one if we agree on this.

Reply:
i see. seastar's `install-dependencies.sh` installs gcc's toolchain.